### ISSUE TITLE
Unreadable text on the Styling Reference page

### ISSUE THREAD

**Issue by TorreyHoffman** [Polymer/docs#284]
At the top of http://www.polymer-project.org/docs/polymer/styling.html
there is pink on fuschia text which is nearly unreadable due to lack of contrast.

Please consider accessibility when choosing colors.  

See http://webaim.org/resources/contrastchecker/ for recommendations.  Contrast ratio should be at least 4.5:1 for normal text.  

The light pink on fuschia is only 1.5:1.

**Comment by arthurevans** [comment]
I believe we fixed this in a batch of contrast fixes.
